### PR TITLE
Add re-usable components

### PR DESCRIPTION
**Pull request by Mukumbuta**
Features implemented:
- Delete all the boilerplate from CRA (text, images, styles) 👍🏾 
- Create a directory called `components`  👍🏾 
- Inside `components`, create a new `Calculator.js` file 👍🏾 
- In `Calculator.js`, create a React component that matches the provided design 👍🏾 
- Add the `Calculator `component 👍🏾 
- Add the `Button `component 👍🏾 
- Add the `Display `component 👍🏾 
- Fixed all linter errors 👍🏾 
- Use class-based components (All components should be class-based) 👍🏾 
- Test it in the browser; it should render without problems 👍🏾

### CODE REVIEW

**Review by jerryowusu — COMMENTED** [review]
Hi @Mukumbuta , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is still using the old readme template. Kindly update your project with the latest [readme template](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (Davidosky007) [inline src/App.js]: * Kindly make all components class-based  as required for this milestone  [Check requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Mukumbuta 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* Descriptive commit messages 👍 
* No linter errors 👍 
* Proper use of gitflow 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Santiago220991 — APPROVED** [review]
Hi @Mukumbuta, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
:heavy_check_mark: Applied changes suggested by the last reviewer.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Followed Gitflow.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**